DQMOffline/Trigger/python/B2GMonitoring_cff.py

```diff
@@ -19,139 +19,157 @@
 # HLT_AK8PFHT900_TrimMass50_v*
 # HLT_AK8PFHT700_TrimR0p1PT0p03Mass50
 
-PFHT1050_Mjjmonitoring = hltMjjmonitoring.clone()
-PFHT1050_Mjjmonitoring.FolderName = cms.string('HLT/B2G/PFHT1050')
+PFHT1050_Mjjmonitoring = hltMjjmonitoring.clone(
+    FolderName = 'HLT/B2G/PFHT1050',
+    #numGenericTriggerEventPSet.hltPaths == ["HLT_PFHT1050_v*"],
+    jets = "ak8PFJetsPuppi",
+    jetSelection = "pt > 200 && eta < 2.4"
+)
 PFHT1050_Mjjmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_PFHT1050_v*")
```

Comment:
```suggestion
PFHT1050_Mjjmonitoring = hltMjjmonitoring.clone(
    FolderName = 'HLT/B2G/PFHT1050',
    jets = "ak8PFJetsPuppi",
    jetSelection = "pt > 200 && eta < 2.4",
    numGenericTriggerEventPSet = dict(hltPaths = ["HLT_PFHT1050_v*"])
)
```
This should work, too (but I haven't tested it). If not, one should at least be able to do
```py
PFHT1050_Mjjmonitoring.numGenericTriggerEventPSet.hltPaths = ["HLT_PFHT1050_v*"]
```
This comment applies to various other places in this PR, but it is optional (obviously, the review is up to DQM experts).

Reply:
Thanks @missirol  indeed the dict solution is the recommended one I believe

Reply:
Okay. Actually, dict was not effective in some previous PRs, and that's why I didn't consider it in this PR. Will consider in this PR. Thank you for the suggestions.